res/controllers/Denon-MC7000.midi.xml

```diff
@@ -2123,23 +2151,23 @@
                 </options>
             </control>
             <control>
-                <group>[Library]</group>
-                <key>track_color_prev</key>
+                <group>[Channel4]</group>
+                <key>MC7000.parameterButtonLeftShifted</key>
                 <description></description>
                 <status>0x97</status>
                 <midino>0x2A</midino>
                 <options>
-                    <normal/>
+                    <script-binding/>
                 </options>
             </control>
```

Comment:
modelling a shifted button as a separate component has the issue that if you press the button, press and hold shift and then release the button, the controller usually does only creates a release event for the shifted button. So the unshifted button stays on. ComponentsJS has some code that works around this but for that the `shift` and `unshift` mechanic needs to be used (see `ComponentContainer.shift` implementation for details). Can you test whether thats a problem here?

Reply:
I don't think this should be an issue since we only handle button down events anyway:

```js
if (value === 0) {
    return;
}
```

Reply:
> I don't think this should be an issue since we only handle button down events anyway:

For these controls yes, though it does make a difference for the hotcue pads for example. Thanks for double checking though.